### ISSUE TITLE
Adding "show" props

### ISSUE THREAD

**Issue by conorhastings** [mapbox/react-colorpickr#36]
Wondering if there would be support for adding "show props" here. For example application that does not support alpha transparency can pass in prop to not show the alpha picker, etc... Thanks!

**Comment by tmcw** [comment]
If we added this it seems like we'd need to add a validation step to make sure the color picker is never initialized with an alpha < 1.

**Comment by conorhastings** [comment]
Yeah agreed that would have to happen and makes this is a little trickier,  but only if the user passes in the option to not show alpha. I'd be happy to work on something/submit a PR and see what you guys think.

**Comment by samanpwbb** [comment]
Would love to see a little more visibility control over UI elements in the picker. Sometimes all you want is a color field.
